doc/tutorials/basic/panes.md

```diff
@@ -0,0 +1,433 @@
+
+::::{tab-set}
+
+:::{tab-item} conda
+:sync: conda
+
+```bash
+conda install -y -c conda-forge altair hvplot matplotlib plotly
+```
+
+:::
+
+:::{tab-item} pip
+:sync: pip
+
+```bash
+pip install altair hvplot matplotlib plotly
+```
+
+:::
+
+::::
+
+## Display Strings
+
+The [`Str`](../../reference/panes/Str.ipynb) pane can display any text.
+
+Run the code:
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+pn.pane.Str(
+    'This is a raw string that will not be formatted in any way.',
+).servable()
+```
+
+:::{note}
+We add `.servable()` to the component to add it to the app served by `panel serve app.py --autoreload`. Adding `.servable()` is not needed to display the component in a notebook.
+:::
+
+:::{note}
+To learn in detail how a pane like `Str` works, refer to its *reference guide*.
+:::
+
+Click [this link](../../reference/panes/Str.ipynb) to the `Str` *reference guide* and spend a few minutes to familiarize yourself with its organization and content.
+
+## Display Markdown
+
+The [`Markdown`](../../reference/panes/Markdown.ipynb) pane can format and display [*markdown*](https://en.wikipedia.org/wiki/Markdown) strings.
+
+Run the code:
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+pn.pane.Markdown("""\
+# Wind Turbine
+
+A wind turbine is a device that converts the kinetic energy of wind into \
+[electrical energy](https://en.wikipedia.org/wiki/Electrical_energy).
+
+Read more [here](https://en.wikipedia.org/wiki/Wind_turbine).
+""").servable()
+```
+
+:::{tip}
+It's key for success with Panel to be able to navigate the [Component Gallery](../../reference/index.md) and use the *reference guides*.
+:::
+
+Click [this link](https://panel.holoviz.org/reference/index.html#panes) to the [Panes Section](https://panel.holoviz.org/reference/index.html#panes) of the [Component Gallery](../../reference/index.md). Identify the [Markdown Reference Guide](../../reference/panes/Markdown.ipynb) and open it. You don't have to spend time studying the details right now.
+
+It should look like
+
+[![Panes Gallery and Markdown Reference Guide](../../_static/images/panes_markdown_reference.png)](https://panel.holoviz.org/reference/index.html#panes)
+
+### Display Alerts
+
+The [`Alert`](../../reference/panes/Alert.ipynb) pane can format and display [*markdown*](https://en.wikipedia.org/wiki/Markdown) strings inside a nicely styled *Alert* pane.
+
+Run the code:
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+pn.pane.Alert("""
+## Markdown Sample
+
+This sample text is from [The Markdown Guide](https://www.markdownguide.org)!
+""", alert_type="info").servable()
+```
+
+## Display Plots
+
+Pick a plotting library below.
+
+:::::{tab-set}
+
+::::{tab-item} Altair
+:sync: altair
+
+Run the code below.
+
+```{pyodide}
+import altair as alt
```

Comment:
The Altair plot does not display if you go to another tab and back to Altair tab.

It displayes after the window is resized though.


https://github.com/holoviz/panel/assets/42288570/8c775d5d-6218-4af8-9754-f50449d52ac2



All other plots works.